static/js/subs.js

```diff
@@ -366,6 +374,10 @@ exports.mark_sub_unsubscribed = function (sub) {
             settings.collapse('hide');
         }
 
+        // Update subscribers count
+        var count = stream_data.get_subscribers_count(sub.name) - 1;
```

Comment:
There's a number of ways to do it (e.g. use a filter() call to remove the others), but basically this just requires iterating over sub.subscribers ...